bundles/org.eclipse.swt/Eclipse SWT PI/gtk/library/atk.c

```diff
@@ -24,11 +24,6 @@
 #define ATK_NATIVE(func) Java_org_eclipse_swt_internal_accessibility_gtk_ATK_##func
 #endif
 
-#ifdef _WIN32
```

Comment:
Why do you remove these ifdef _WIN32? While I don't say they should be preserved these are clearly noise in this patch.

Reply:
Agreed, not removed on purpose. They were removed when I built the bindings using the build.sh. I used the latest release but not the latest I build. Is my SWTTools out of date? How can I prevent them from being removed?

Reply:
Use the latest I-build SWTTools always, that should fix the issue.

Reply:
Roger, will fix within a day.

Reply:
Fixed, check latest commit